test/correctness/rfactor.cpp

```diff
@@ -17,27 +17,59 @@ typedef map<string, vector<string>> CallGraphs;
 
 class CheckCalls : public IRVisitor {
```

Comment:
It looks like this CheckCalls mutator (and related helper functions) should be moved to a header to reduce the duplicated code. See https://github.com/halide/Halide/blob/master/test/common/gpu_object_lifetime.h for an example (used by several tests).